tests/test_frequenz_dispatch.py

```diff
@@ -15,18 +15,30 @@
 from frequenz.client.dispatch.test.client import FakeClient, to_create_params
 from frequenz.client.dispatch.test.generator import DispatchGenerator
 from frequenz.client.dispatch.types import Dispatch as BaseDispatch
-from frequenz.client.dispatch.types import Frequency
+from frequenz.client.dispatch.types import DispatchEvent as BaseDispatchEvent
+from frequenz.client.dispatch.types import Event, Frequency, RecurrenceRule
 from pytest import fixture
 
-from frequenz.dispatch import Created, Deleted, Dispatch, DispatchEvent, Updated
+from frequenz.dispatch import (
+    Created,
+    Deleted,
+    Dispatch,
+    DispatchEvent,
+    RunningState,
+    Updated,
+)
 from frequenz.dispatch.actor import DispatchingActor
 
 
-# This method replaces the event loop for all tests in the file.
-@fixture
-def event_loop_policy() -> async_solipsism.EventLoopPolicy:
-    """Return an event loop policy that uses the async solipsism event loop."""
-    return async_solipsism.EventLoopPolicy()
+@fixture(scope="session")
```

Comment:
:warning:  This shouldn't be necessary and I really think we shouldn't go this way. Things should work if every test runs in its own loop and if it doesn't it means some cleanup is missing or messed up. :warning: 

Oh, I just noticed it also goes back from the correct `event_loop_policy()` to the old (and deprecated) `event_loop()` which is supposed to stop working soon :tm:. This is definitely not an option.

Reply:
I am happy to take any working alternative ;)